pkg/providers/ovirt/templates/template-finder.go

```diff
@@ -99,12 +99,18 @@ func (f *TemplateFinder) getTemplate(os string, workload string) (*templatev1.Te
 }
 
 // GetLabels fetches OS and workload specific labels
-func (f *TemplateFinder) GetLabels(vm *ovirtsdk.Vm) (map[string]string, error) {
+func (f *TemplateFinder) GetLabels(template *templatev1.Template, vm *ovirtsdk.Vm) (map[string]string, map[string]string, error) {
```

Comment:
GetLabels returns both labels and annotations - that is slightly confusing:
I guess we don't want to break this into two methods not to repeat the OS resolving, I'd just change the name to be more precise, i.e. GetMetadata